documentation/hpcc-tf-for-developers.md

```diff
@@ -0,0 +1,373 @@
+# For Developers: Tutorial of HPCC Easy Deploy Terraform 
+
+This tutorial explains the terraform that deploys HPCC Systems on an azure kubernetes service (aks). The terraform was designed to enable one to deploy HPCC Systems easily.
```

Comment:
Minor: Ensure 'terraform', 'kubernetes', and 'github' -- when used as product names like they are here -- are capitalized in all external documentation.

Reply:
fixed